README.md

```diff
@@ -77,6 +77,7 @@ See the [Wiki](https://github.com/hasadna/anyway/wiki/Setting-up-a-Python-develo
    days. Be prepared.
 1. Populate United Hatzalah sample data: `python main.py process united --light` for the complete,
    or more recent data please contact the Anyway team.
+1. Populate LMS vehicles registered in cities : `python main.py process registered`: this will take less than an hour
```

Comment:
Please give the parser more a informative name

Reply:
This is what it does, it import the vehicles registered in cities tables of LMS. Actually I think it better then the 'Populate the data" we have earlier in the file (CBS process)

Reply:
I think Roy was referring to the name `registered` - perhaps `registered_vehicles` would be more informative.

Reply:
ok by me :) will change.